lib/medium-admin.php

```diff
@@ -136,6 +136,12 @@ public static function add_meta_boxes_post($post) {
   public static function save_post($post_id, $post) {
     if (defined('DOING_AUTOSAVE') && DOING_AUTOSAVE) return;
 
+    $allowed_post_types = (array) apply_filters("medium_allowed_post_types", array("post"));
```

Comment:
Thanks very much for working through all this, much appreciated. Let's drop the cast to (array) - the standard requirement of filters is that they return the value untouched if they don't need to modify it, and it's hard to imagine someone converting an array to, say, an object just for the hell of it.